src/uu/factor/Cargo.toml

```diff
@@ -14,7 +14,11 @@ edition = "2018"
 [lib]
 path = "src/factor.rs"
 
+[build-dependencies]
+num-traits="0.2"
+
 [dependencies]
+num-traits="0.2"
```

Comment:
I’d prefer that you put spaces before and after the `=`.

Reply:
Done